.vscode/launch.json

```diff
@@ -8,6 +8,8 @@
       "runtimeExecutable": "${execPath}",
       "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
       "outFiles": ["${workspaceFolder}/out/**/*.js"],
+      "stopOnEntry": false,
```

Comment:
Locally, I get a warning message saying "Property stopOnEntry is not allowed." Would it be possible to remove this, or is this important for development on your end?

<img width="587" alt="image" src="https://github.com/user-attachments/assets/150d7d62-c097-4a34-bb11-2154fd21c1b8">


```suggestion
```

Reply:
yes, sorry, I put this for test purpose of this extension only. I will remove it from PR